source/backend/api/Services/PropertyService.cs

```diff
@@ -72,5 +89,12 @@ public PimsProperty Update(PimsProperty property)
 
             return GetById(newProperty.Internal_Id);
         }
+
+        private Point TransformCoordiates(Geometry location)
+        {
+            // return property spatial location in lat/long (4326)
+            var newCoords = _coordinateService.TransformCoordinates(SpatialReference.BCALBERS, SpatialReference.WGS84, location.Coordinate);
```

Comment:
Yeah, I think this is fair for now that we are always going to be going from albers to wgs84?